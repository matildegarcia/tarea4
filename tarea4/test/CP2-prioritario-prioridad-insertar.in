# implementar prioritaria, prioridad, insertarEnCP

crearColaPrioridad 25

# creo Jugador 5
crearPartida
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 5 21 Uruguay Juan
imprimirJugador

# inserto jugador en cola
insertarColaPrioridad

# creo Jugador 25 
crearPartida
crearJugada
1 1 1 1 1
agregarEnPartida
crearJugada
2 2 2 2 2 
agregarEnPartida
crearJugada
3 2 3 4 4
agregarEnPartida
crearJugador 25 18 Uruguay Luis
imprimirJugador

insertarColaPrioridad

# el jugador prioritario es el 25
imprimirJugadorPrioritario

# creo Jugador 11
crearPartida
crearJugada
1 4 1 4 1
agregarEnPartida
crearJugada
2 2 3 2 3 
agregarEnPartida
crearJugada
3 2 1 4 2
agregarEnPartida
crearJugador 11 21 Uruguay Nicolas
imprimirJugador

# inserto jugador en cola
insertarColaPrioridad

# el jugador prioritario sigue siendo 25
imprimirJugadorPrioritario

# se imprime la edad de jugador 5
imprimirPrioridadJugador 5

# Jugador 5 est치 en la cola
estaJugadorEnColaPrioridad 5

# creo Jugador 6 e inserto en cola
crearPartida
crearJugada
1 1 2 3 4
agregarEnPartida
crearJugador 6 16 Uruguay Juan
imprimirJugador
insertarColaPrioridad

# ahora el jugador prioritario es el 6
imprimirJugadorPrioritario

# la edad del jugador 6 es la de mayor prioridad
imprimirPrioridadJugador 6

# creo Jugador 8 como mas prioritario y agrego a cola
crearPartida
crearJugada
1 4 1 3 1
agregarEnPartida
crearJugada
2 2 1 2 3 
agregarEnPartida
crearJugada
3 2 3 1 1
agregarEnPartida
crearJugador 8 14 Argentina Martin
imprimirJugador
insertarColaPrioridad

# creo Jugador 7 y agrego a cola
crearPartida
crearJugada
1 1 2 4 4
agregarEnPartida
crearJugador 7 19 Uruguay Jessica
imprimirJugador
insertarColaPrioridad

# creo Jugador 10 y agrego a cola
crearPartida
crearJugada
1 4 2 4 4
agregarEnPartida
crearJugador 10 15 Uruguay Vanessa
imprimirJugador
insertarColaPrioridad

# creo Jugador 1 y agrego a cola
crearPartida
crearJugada
1 1 2 4 2
agregarEnPartida
crearJugador 1 20 Espa침a Santiago
imprimirJugador
insertarColaPrioridad

# creo Jugador 2 y agrego a cola
crearPartida
crearJugada
1 1 3 3 2
agregarEnPartida
crearJugador 2 25 Mexico Matias
imprimirJugador
insertarColaPrioridad

# creo Jugador 9 y agrego a cola
crearPartida
crearJugada
1 1 4 4 3
agregarEnPartida
crearJugador 9 26 Turquia Florencia
imprimirJugador
insertarColaPrioridad

# creo Jugador 3 y agrego a cola
crearPartida
crearJugada
1 4 4 4 2
agregarEnPartida
crearJugador 3 29 Holanda Francisco
imprimirJugador
insertarColaPrioridad

# creo Jugador 4 y agrego a cola
crearPartida
crearJugada
1 2 2 2 2
agregarEnPartida
crearJugador 4 21 Ecuador Federico
imprimirJugador
insertarColaPrioridad

# ahora el jugador prioritario es el 8
imprimirJugadorPrioritario

# la edad del jugador 8 es la de mayor prioridad
imprimirPrioridadJugador 8

# verifico que el jugador 1 est치 en la cola
estaJugadorEnColaPrioridad 1

# la edad del jugador 1
imprimirPrioridadJugador 1

# verifico que el jugador 10 est치 en la cola
estaJugadorEnColaPrioridad 10

# la edad del jugador 10
imprimirPrioridadJugador 10

liberarColaPrioridad

Fin
