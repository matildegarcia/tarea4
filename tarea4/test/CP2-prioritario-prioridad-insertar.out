1>#  implementar prioritaria, prioridad, insertarEnCP.
2>3>#  creo Jugador 5.
4>5>6>7>8>Jugador 5: Juan, Uruguay, 21 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
9>#  inserto jugador en cola.
10>11>#  creo Jugador 25 .
12>13>14>15>16>17>18>19>20>Jugador 25: Luis, Uruguay, 18 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugada: 2 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
Jugada: 3 
Movimientos: Izquierda Arriba Abajo Abajo 
21>22>#  el jugador prioritario es el 25.
23>Jugador 25: Luis, Uruguay, 18 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugada: 2 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
Jugada: 3 
Movimientos: Izquierda Arriba Abajo Abajo 
24>#  creo Jugador 11.
25>26>27>28>29>30>31>32>33>Jugador 11: Nicolas, Uruguay, 21 años
Jugada: 1 
Movimientos: Abajo Derecha Abajo Derecha 
Jugada: 2 
Movimientos: Izquierda Arriba Izquierda Arriba 
Jugada: 3 
Movimientos: Izquierda Derecha Abajo Izquierda 
34>#  inserto jugador en cola.
35>36>#  el jugador prioritario sigue siendo 25.
37>Jugador 25: Luis, Uruguay, 18 años
Jugada: 1 
Movimientos: Derecha Derecha Derecha Derecha 
Jugada: 2 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
Jugada: 3 
Movimientos: Izquierda Arriba Abajo Abajo 
38>#  se imprime la edad de jugador 5.
39>La prioridad del jugador es 21.
40>#  Jugador 5 está en la cola.
41>El jugador 5 está en la cola de prioridad.
42>#  creo Jugador 6 e inserto en cola.
43>44>45>46>47>Jugador 6: Juan, Uruguay, 16 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
48>49>#  ahora el jugador prioritario es el 6.
50>Jugador 6: Juan, Uruguay, 16 años
Jugada: 1 
Movimientos: Derecha Izquierda Arriba Abajo 
51>#  la edad del jugador 6 es la de mayor prioridad.
52>La prioridad del jugador es 16.
53>#  creo Jugador 8 como mas prioritario y agrego a cola.
54>55>56>57>58>59>60>61>62>Jugador 8: Martin, Argentina, 14 años
Jugada: 1 
Movimientos: Abajo Derecha Arriba Derecha 
Jugada: 2 
Movimientos: Izquierda Derecha Izquierda Arriba 
Jugada: 3 
Movimientos: Izquierda Arriba Derecha Derecha 
63>64>#  creo Jugador 7 y agrego a cola.
65>66>67>68>69>Jugador 7: Jessica, Uruguay, 19 años
Jugada: 1 
Movimientos: Derecha Izquierda Abajo Abajo 
70>71>#  creo Jugador 10 y agrego a cola.
72>73>74>75>76>Jugador 10: Vanessa, Uruguay, 15 años
Jugada: 1 
Movimientos: Abajo Izquierda Abajo Abajo 
77>78>#  creo Jugador 1 y agrego a cola.
79>80>81>82>83>Jugador 1: Santiago, España, 20 años
Jugada: 1 
Movimientos: Derecha Izquierda Abajo Izquierda 
84>85>#  creo Jugador 2 y agrego a cola.
86>87>88>89>90>Jugador 2: Matias, Mexico, 25 años
Jugada: 1 
Movimientos: Derecha Arriba Arriba Izquierda 
91>92>#  creo Jugador 9 y agrego a cola.
93>94>95>96>97>Jugador 9: Florencia, Turquia, 26 años
Jugada: 1 
Movimientos: Derecha Abajo Abajo Arriba 
98>99>#  creo Jugador 3 y agrego a cola.
100>101>102>103>104>Jugador 3: Francisco, Holanda, 29 años
Jugada: 1 
Movimientos: Abajo Abajo Abajo Izquierda 
105>106>#  creo Jugador 4 y agrego a cola.
107>108>109>110>111>Jugador 4: Federico, Ecuador, 21 años
Jugada: 1 
Movimientos: Izquierda Izquierda Izquierda Izquierda 
112>113>#  ahora el jugador prioritario es el 8.
114>Jugador 8: Martin, Argentina, 14 años
Jugada: 1 
Movimientos: Abajo Derecha Arriba Derecha 
Jugada: 2 
Movimientos: Izquierda Derecha Izquierda Arriba 
Jugada: 3 
Movimientos: Izquierda Arriba Derecha Derecha 
115>#  la edad del jugador 8 es la de mayor prioridad.
116>La prioridad del jugador es 14.
117>#  verifico que el jugador 1 está en la cola.
118>El jugador 1 está en la cola de prioridad.
119>#  la edad del jugador 1.
120>La prioridad del jugador es 20.
121>#  verifico que el jugador 10 está en la cola.
122>El jugador 10 está en la cola de prioridad.
123>#  la edad del jugador 10.
124>La prioridad del jugador es 15.
125>126>Fin.
